### PR TITLE
Fix dead link in Kubernetes on Mesos getting started guide

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

**Comment by nak3** [comment]
If you prefer, I will squash pull/11868, pull/11127 and pull/11125

File: docs/getting-started-guides/mesos.md [hunk 1]
```diff
@@ -362,7 +362,7 @@ Future work will add instructions to this guide to enable support for Kubernetes
 [4]: ../../cluster/addons/dns/README.md
 [5]: http://open.mesosphere.com/getting-started/cloud/google/mesosphere/
 [6]: http://mesos.apache.org/
-[7]: https://github.com/mesosphere/kubernetes-mesos/blob/master/docs/issues.md
+[7]: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/contrib/mesos/docs/issues.md
```
Comment (mikedanese) [inline docs/getting-started-guides/mesos.md]: can you use a relative link so we can version it?

**Comment by wojtek-t** [comment]
@k8s-bot ok to test

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**